capstone-frontend/src/components/UserCard.jsx

```diff
@@ -7,36 +7,78 @@ import '../styles/Groups.css';
 class UserCard extends Component {
   removeMember = () => {
     const jsonBody = {
-      "remove_memberIDs": this.props.user.id, 
+      "remove_memberIDs": this.props.user.id,
       "id": this.props.club.id
     }
-    fetch(`/api/clubs`, {
-        method: "put", 
-        body: JSON.stringify(jsonBody)
-      })
+    fetch(`/api/clubs`, { method: "put", body: JSON.stringify(jsonBody) })
       .then(this.props.removeMember(this.props.user.id))
+      .catch(function (err) {
+        //TODO #61: Centralize error output
+        alert(err);
+      });
+  }
+
+  acceptMember = () => {
```

Comment:
What happens if this request somehow gets sent without the user being in the requestIDs array first before adding them to the memberIDs list?

Reply:
Good catch! In this case the backend won't throw an error, as Firebase `FieldValue.arrayRemove` does not throw errors on non-existence of the item it looks to remove. So the user will be added as a member, despite not being a requester. 
I'll add a check before the fetch.